packages/flutter_test/lib/src/binding.dart

```diff
@@ -1776,3 +1779,44 @@ class _MockHttpHeaders extends HttpHeaders {
   @override
   String value(String name) => null;
 }
+
+/// Helper class that mocks assets message channel if platform environment
+/// variable exists.
+class _AssetMocker {
+  _AssetMocker();
+
+  static Set<dynamic> _allowedKeys;
+
+  /// Mocks message channel.
+  ///
+  /// This function will mock message channel or override existing mock every
+  /// time it is called.
+  static void mock() {
```

Comment:
We should call this something other than mock. `mockFlutterAssets`?

Reply:
Actually, since all fields are static I would remove the class entirely and place them under the test bindings.

Reply:
fixed